sycl/test/basic_tests/vector_cross.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %RUN_ON_HOST %t.out
```

Comment:
Is there any motivation to limit testing to host only?

Reply:
I guess, the reason is to avoid having dependency on runtime in the in-tree test.

Reply:
Sounds reasonable.